### DISCUSSION TITLE
How do I implement an options which can be specified multiple times?

### DISCUSSION THREAD

**Discussion by ws1088** [pytest-dev/pytest#8601] [Q&A] [answered]
Hi All,
Is it possible to create my option, say `--my_opt` so that I can specify it multiple times? For example:
```
pytest --my_opt abc --my_opt def
```
and I can retrieve the values in order, like
```
['abc', 'def']
```
Thanks in advance!

**Comment by RonnyPfannschmidt** [comment, marked as answer]
I believe the append action helps with that

**Reply by ws1088** [reply]
thanks, i confirm that's the solution. thanks again!

**Reply by nicoddemus** [reply]
@ws1088 could you please mark it as the answer to help others? Thanks!

**Comment by nicoddemus** [comment]
Here's how pytest defines `-v`:

https://github.com/pytest-dev/pytest/blob/2049ae271e0690279a029b62791cf387a2d94597/src/_pytest/terminal.py#L116-L122

**Reply by ws1088** [reply]
thanks. `action="count"` will count how many times i enter `-v`, which is not what I am looking for. but nevertheless, good to know, thanks!

**Reply by nicoddemus** [reply]
Ahh sorry I read it only at a glance.
